### ISSUE TITLE
Setting new values in the debugger

### ISSUE THREAD

**Issue by m0ppers** [BartmanAbyss/vscode-amiga-debug#257]
I am unsure if this is a bug or just a missing feature.

When I pause execution and try to modify variables in the debugger I am getting the following error:

`Could not continue: Variable object not found (from var-assign y 20)`

This happens for all variables. Global and Local. Bug or missing feature? I have tried it under Windows and Linux and the result is the same.

**Comment by BartmanAbyss** [comment]
yeah, seems like a bug. Unfortunately won't be able to fix anytime soon due to health reasons.
